tests/integrationv2/configuration.py

```diff
@@ -95,3 +98,248 @@
 
 # List of ports available to tests.
 available_ports = AvailablePorts()
+
+
+
+# Server certificates used to test matching domain names client with server_name
+# ( cert_path, private_key_path, domains[] )
+SNI_CERTS = {
```

Comment:
The indentation makes this a bit hard to follow

Reply:
Fixed the indentation to me less eye destroying.

Reply:
Much better!